src/App.js

```diff
@@ -2,14 +2,33 @@ import React from "react";
 import { Switch, Redirect, Route, withRouter } from "react-router-dom";
 import AuthenticationRoute from "./containers/AuthenticationRoute";
 import routes from "./helpers/routes";
+import { CURRENCY } from "./config";
 import LoginScreen from "./components/LoginScreen";
+import Currency from "./components/Currency";
+
 
 const App = props => {
-  const isAuthenticated = true;
+  const isAuthenticated = false;
+  const currencyValue = 1500015646;
+  const currency = CURRENCY;
```

Comment:
This is not the way we intend to use CURRENCY. The whole idea is having currency to work as a default.